bare/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+type buildbarnProcess struct {
+	config string
+	binary string
+}
+
+func bbStart(bbProcess buildbarnProcess) (*os.Process, error) {
+	path, found := bazel.FindBinary(path.Join("cmd", bbProcess.binary), bbProcess.binary)
+	configPath, err := bazel.Runfile(bbProcess.config)
+	if !found {
+		return nil, fmt.Errorf("Couldn't find %s", bbProcess.binary)
+	}
+	if err != nil {
+		return nil, err
+	}
+	cmd := exec.Command(path, configPath)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Process, cmd.Start()
```

Comment:
Maybe it's a bit nicer to just call `cmd.Run()` instead and `log.Fatal()` in case `err != nil`?